embedded/ahtree/ahtree.go

```diff
@@ -31,6 +32,7 @@ import (
 )
 
 var ErrIllegalArguments = errors.New("illegal arguments")
```

Comment:
wdyt about putting all these vars in a single block? 
```
var (

)
```

Reply:
I think it could be done, probably it another PR. Inline var declaration sometimes make it easier to lookup for error declaration without IDE support, just that